docs/Lua API.rst

```diff
@@ -4079,10 +4079,22 @@ DFHack core invokes the scripts in the *core context* (see above);
 however it is possible to call them from any lua code (including
 from other scripts) in any context, via the same function the core uses:
 
-* ``dfhack.run_script(name[,args...])``
-
+* ``dfhack.run_script(command[, ...])``
+   
   Run a lua script in hack/scripts/, as if it was started from dfhack command-line.
-  The ``name`` argument should be the name stem, as would be used on the command line.
+  The syntax for the argument is similar to dfhack.run_command.
+  The command can be passed as a table, multiple string
+  arguments, or a single string argument (not recommended - in this case, the
+  usual DFHack console tokenization is used).
+  The following examples are equivalent::
+        dfhack.run_script({'ls', '-a'})
+        dfhack.run_script('ls', '-a')
+        dfhack.run_script('ls -a')  -- not recommended
```

Comment:
Looks like I forgot to explain this, sorry. `run_script` and `run_command` are not similar, and only the second example you have here is valid syntax. It won't actually work either, because `ls` isn't a Lua script (it's a built-in command). That's why I suggested the `weather` example, which _is_ a Lua script, and it's also fairly simple and unlikely to change in the future.